src/theme/breadcrumbs.js

```diff
@@ -0,0 +1,54 @@
+if ($('#wikiBreadcrumb ol.breadcrumb')) {
+    var here = location.href.replace(/(\?.*)$/,'').split('/').slice(3);
+
+    var parts = [{
+        text: "Foreword",
+        link: "/"
+    }];
+
+    for (var j = 0; j < here.length; j++) {
+        var part = here[j];
+
+        switch (part.toLowerCase()) {
+            case 'bdscript':
+                var pageName = 'Functions';
+                break;
+            case 'guides':
+                var pageName = 'Guides';
+                break;
+            case 'resources':
+                var pageName = 'Resources';
+                break;
+            case 'callbacks':
+                var pageName = 'Callbacks';
+                break;
+            case 'premium':
+                var pageName = 'Premium';
+                break;
+            case 'javascript':
+                var pageName = 'JavaScript';
+                break;
+        };
+
+        if (pageName == undefined) {
+            pageName = document.title.split('-')[0];
+        };
+
+
+        var link = '/' + here.slice(0, j + 1).join('/');
+
+        if (part == 'guides' || part == 'resources' || part == 'bdscript' || part == 'premium' || part == 'javascript' || part == 'callbacks') {
```

Comment:
It can fail because `part` does not have to be in lowercase.
Addesing the previous comment, you can use [`Object.keys()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) and then [`.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) on the resulting array.